src/Components/test/E2ETestMigration/Tests/InputFileTest.cs

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BasicTestApp;
+using BasicTestApp.FormsTest;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+using Xunit.Abstractions;
+using System.Threading.Tasks;
+using PlaywrightSharp;
+using Microsoft.AspNetCore.BrowserTesting;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests
+{
+    public class InputFileTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
+    {
+        private string _tempDirectory;
+
+        public InputFileTest(
+            ToggleExecutionModeServerFixture<Program> serverFixture,
+            ITestOutputHelper output)
+            : base(serverFixture, output)
+        {
+        }
+
+        private IPage _page;
+        private IBrowserContext _browser;
+
+        protected override async Task InitializeCoreAsync(TestContext context)
+        {
+            await base.InitializeCoreAsync(context);
+
+            _tempDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("N"));
+            Directory.CreateDirectory(_tempDirectory);
+
+            //Navigate(ServerPathBase, noReload: _serverFixture.ExecutionMode == ExecutionMode.Client);
+
+            _browser = await BrowserManager.GetBrowserInstance(BrowserKind.Chromium, BrowserContextInfo);
+            var page = await _browser.NewPageAsync();
+            var url = _serverFixture.RootUri + "subdir";
+            var response = await page.GoToAsync(url);
+
+            Assert.True(response.Ok, "Got: " + response.StatusText + "from: " + url);
+            Output.WriteLine("Loaded page");
+
+            await MountTestComponentAsync<InputFileComponent>(page);
```

Comment:
TODO: Lift MountTestComponentAsync<T>/initialize core to ComponentTestBase and allow just specifying component to mount, need to double check if any tests do anything special